dockers/docker-orchagent/switch.json.j2

```diff
@@ -12,7 +12,8 @@
     {
         "SWITCH_TABLE:switch": {
             "ecmp_hash_seed": "{{ hash_seed }}",
-            "lag_hash_seed": "{{ hash_seed }}"
+            "lag_hash_seed": "{{ hash_seed }}",
+            "fdb_aging_time": "300"
```

Comment:
-> 600 as default.

Reply:
I am fine to change it to 600 seconds to be on the safer side comparing to ARP update interval.